Gemfile

```diff
@@ -8,8 +8,8 @@ group :development do
   gem "rspec"
   gem "rdoc"
   gem "rake"
-  gem "rubocop", "~> 0.71.0", require: false
-  gem "rubocop-performance", require: false
+  gem "rubocop", "~> 0.74.0", require: false
+  gem "rubocop-performance", "~> 1.3.0", require: false
   gem "rubocop-rails", require: false
```

Comment:
Hmm, It may be preferable to specify RuboCop Rails 2.0.0.
https://github.com/codeclimate/codeclimate-rubocop/blob/channel/rubocop-0-74/Gemfile.lock#L51-L55